### PR TITLE
Update to Mentoring FAQ

### PR DESCRIPTION
**Pull request by rahulkeerthi**
- Added in new questions and sections
- Addresses mentor removal question in [exercism repo](https://github.com/exercism/exercism/issues/5203)
- Aligned to proposed changes in general FAQ

### CODE REVIEW

File: pages/mentoring_faqs.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+
+## Better Mentorship
+
 ### What should I do if I don't know how to give feedback on a solution?
 If you're not sure what feedback to give, paste a link to the solution (e.g. `https://exercism.io/mentor/solutions/...`) on the relevant language channel on Slack and brainstorm some ideas with other mentors.
 
 ### How can I improve my mentoring skills?
-This is something we're extremely passionate about helping you with and we are in the process of writing some awesome guides on giving great feedback with lots of examples. Eventually we hope to have a guide for each track.
+This is something we're extremely
+passionate about helping you with and we are in the process of writing some awesome guides on giving great feedback with lots of examples. Eventually we hope to have a guide for each track.
```
Comment (ErikSchierboom) [inline pages/mentoring_faqs.md:24]: Was the cutoff from extremely to passionate on different lines intentional?

Comment (rahulkeerthi) [reply pages/mentoring_faqs.md:24]: Oh, no, that's a typo!

Comment (rahulkeerthi) [reply pages/mentoring_faqs.md]: Fixed now!

File: pages/mentoring_faqs.md [hunk 2]
```diff
@@ -1,11 +1,21 @@
-Here we have collated a list of the questions asked frequently by mentors.
+Here we have collated a list of the questions asked frequently by mentors. If you have a question that hasn't been answered here, please post it on the **\#questions** channel on Slack.
```
Comment (kotp) [inline pages/mentoring_faqs.md:1]: Should we make this a clickable link?

```suggestion
Here we have collated a list of the questions asked frequently by mentors. If you have a question that hasn't been answered here, please post it on the **[\#questions](https://exercism-team.slack.com/archives/CAQP7JL3T)** channel on Slack.
```

**Pull request merged**